README.md

```diff
@@ -97,3 +97,64 @@ mvn spring-boot:run
 
 Swagger docs in JSON: http://localhost:8080/v3/api-docs
 Swagger UI: http://localhost:8080/swagger-ui/index.html?url=/v3/api-docs
+
+
+## Gremlin Debugging
+
+Gremlin tools can be found on the [Apache Tinkerpop website](https://tinkerpop.apache.org/).
+
+### Gremlin Console
+
+The console can interface directly with our Gremlin Cosmos instance. Database credentials need to be saved in
+a yaml file. Eg:
+
+```yaml
+hosts: [$gremlin-endpoint$.gremlin.cosmosdb.azure.com]
+port: 443
+username: /dbs/$db-name$/colls/$graph-name$
+password: $password$
+connectionPool: {
+  enableSsl: true}
+serializer: { className: org.apache.tinkerpop.gremlin.driver.ser.GraphSONMessageSerializerV1d0, config: { serializeResultToString: true }}
+```
+
+This file then needs to be referenced from a command line argument. The following example was made with Powershell but a gremlin.sh fil
+also exists in the Gremlin console download.
+
+```powershell
+PS C:\gremlin> .\bin\gremlin.bat
+
+         \,,,/
+         (o o)
+-----oOOo-(3)-oOOo-----
+plugin activated: tinkerpop.server
+plugin activated: tinkerpop.utilities
+plugin activated: tinkerpop.tinkergraph
+
+// double slash is a comment in gremlin
+// connect to remote
+gremlin> :remote connect tinkerpop.server conf/cassemanagement.yaml
+==>Configured $gremlin-endpoint$.gremlin.cosmosdb.azure.com
+gremlin>
```

Comment:
Might want to remove some of the gunk from this section

Reply:
wtf, no? This is the most important part.